Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Email_Notification.yml

```diff
@@ -306,7 +260,12 @@ inputs:
   required: true
   description: Body of the notification (email or ticket) sent to the potential service owner.
   playbookInputQuery:
+- key: RemediationGuidance
+  value: {}
+  required: true
```

Comment:
Does this need to be required?

Reply:
In the "Cortex ASM - Remediation Guidance" playbook, we set the RemediationGuidance key regardless if the Remediation guidance is available or not. So, we will always have this key set. Hence, made this required for the Email/Jira/ServiceNow playbooks